src/components/expanded-state/swap-details/SwapDetailsFeeRow.js

```diff
@@ -21,7 +21,10 @@ export default function SwapDetailsUniswapRow({ tradeDetails, network }) {
     rainbowFeeNative &&
     convertAmountToNativeDisplay(rainbowFeeNative, nativeCurrency);
   const rainbowFeePercentageDisplay = convertAmountToPercentageDisplayWithThreshold(
```

Comment:
maybe is easier to show 0 and not use `convertAmountToPercentageDisplayWithThreshold` if there's no fee? so we don't have to modify convertAmountToPercentageDisplayWithThreshold too